sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/cassandra/LargeITCluster/cassandra-statefulset.yaml

```diff
@@ -0,0 +1,113 @@
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+apiVersion: "apps/v1beta1"
+kind: StatefulSet
+metadata:
+  name: cassandra
+spec:
+  serviceName: cassandra
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: cassandra
+    spec:
+      containers:
+      - name: cassandra
+        image: us.gcr.io/sincere-nirvana-150207/docker-cassandra
```

Comment:
Hi Stephen,

we have successfully used Quay image as well as docker hub Cassandra image for using remote cqlsh. We have updated the scripts for start, stop, data-load.
Basically we did point (1) "1.find a second docker image that contains those tools and have them connect to the cassandra cluster in the other kubernetes pods. You can even load it into the k8s cluster as a singleton pod at the beginning of data-load.sh, and then remove it from the cluster"
Removed custom docker image and its related files